jaeger-core/src/test/java/com/uber/jaeger/TracerTest.java

```diff
@@ -66,6 +68,26 @@ public void testBuildSpan() {
   }
 
   @Test
+  public void testBuildServerSpan() {
+    Span span = (com.uber.jaeger.Span) tracer.buildSpan("flexo")
+                                             .withTag(Tags.SPAN_KIND.getKey(), Tags.SPAN_KIND_SERVER)
```

Comment:
can you put `tracer.buildSpan("flexo")` on the 2nd line to avoid the insane indenting by the formatter? Or at least put `.buildSpan()` on the second line.